linux/scripts/version.sh

```diff
@@ -3,7 +3,7 @@
 version()
 {
     if [[ -z "${TIER}" ]]; then
```

Comment:
When can `${TIER}` be empty?

Reply:
The scripts are a bowl of spaghetti. I saw from `linux/scripts/reconf.sh`:

```
. $(dirname "$0")/version.sh
```